cogs/template.py

```diff
@@ -1,27 +1,29 @@
-import os
-import sys
+# import os

```

Comment:
We don't want this command to show up in the slash commands menu, so it is commented out